packages/site/src/data/vue/podcasts.ts

```diff
@@ -17,7 +17,7 @@ export const podcasts: Podcast<typeof podcastTags[number]>[] = [
 	{
 		title: "Views on Vue",
 		image:
-			"https://assets.fireside.fm/file/fireside-images/podcasts/images/4/4b532b8f-dc99-4b48-90f1-023bd6c0cd90/cover_medium.jpg?v=0",
+			"https://topenddevs.nyc3.digitaloceanspaces.com/0ed14mnygu67dbicn1la5anwx4vp?response-content-disposition=inline%3B%20filename%3D%22viewsonvue-1-scaled.jpeg%22%3B%20filename%2A%3DUTF-8%27%27viewsonvue-1-scaled.jpeg&response-content-type=image%2Fjpeg&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=STO4LZNRL5TU6SLCP4ZY%2F20221027%2Funused%2Fs3%2Faws4_request&X-Amz-Date=20221027T084620Z&X-Amz-Expires=300&X-Amz-SignedHeaders=host&X-Amz-Signature=e23a68acca82ef92d05515f37a995fd29c6d93ff87f33b13456993d7cd36f7f7",
```

Comment:
This link no longer works. 

If you try to go to the link, you will now receive this message
<img width="634" alt="Screen Shot 2022-10-27 at 8 12 17 AM" src="https://user-images.githubusercontent.com/67210629/198328648-3dbbdeb5-a9f8-43f6-806e-16aa8c8655e2.png">

Try this one and see if it works better
```suggestion
			"https://is1-ssl.mzstatic.com/image/thumb/Podcasts126/v4/e1/ee/cd/e1eecd5c-7c90-05e7-19d2-9a2476d82c54/mza_2587166457306830649.jpg/626x0w.webp",
```

Reply:
Thank you, I have done the required change